net/acme-common/files/acme.init

```diff
@@ -9,7 +9,7 @@ HOOK=/usr/lib/acme/hook
 LOG_TAG=acme
 
 # shellcheck source=net/acme/files/functions.sh
-. /usr/lib/acme/functions.sh
+. "$IPKG_INSTROOT/usr/lib/acme/functions.sh"
```

Comment:
I don't think we should have this variable check in the init script.

Since we've removed that START variable anyway, how about just bailing out before the include if the parameter is `enable`? I believe that should fix the issue we were seeing...

Reply:
I don't think I understand. The image builder will run the init script, although I'm not sure with what arguments. Why only check for `enable`?

If I understand it correctly, the image builder sets `IPKG_INSTROOT`, so `source` should find the `functions.sh` file, and in runtime, `IPKG_INSTROOT` is empty, `source` should also find the `functions.sh` file. Isn't that how it should be fixed?

Reply:
Yeah, it will, I just think it's ugly to leak that kind of package manager-specific variables into an init script.

However, it seems to be used in a number of other init files as well, so okay, let's go with that solution...